.github/workflows/features.yml

```diff
@@ -0,0 +1,20 @@
+name: cfg-check
+
+on:
+  pull_request:
+  merge_group:
+  push:
+    branches: [main]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+    cfg-check:
```

Comment:
lets put this in the ci workflow

Reply:
wdym? suggestion pls

Reply:
put it in https://github.com/paradigmxyz/reth/blob/main/.github/workflows/lint.yml and add it to `needs` for `lint-success`

Reply:
thanks